server/database/schema.js

```diff
@@ -0,0 +1,11 @@
+const { Schema, model } = require('mongoose');
+
+const placesSchema = new Schema({
```

Comment:
This is interesting! It looks like right now you're pulling 12 random ID's, different with each call to the database, for a particular stay. I wonder if the database should store a stayId column, and an 'ids' column that has 12 random stayId's that are the 'more places' for that stay. This way the alternate stays that show up for each stay would be persistent.

Reply:
I like that idea. That's something I'll definitely implement before too long.